sigpyproc/FourierSeries.py

```diff
@@ -288,7 +287,8 @@ def toFile(self, filename=None):
         if filename is None:
             filename = f"{self.header.basename}.spec"
         outfile = self.header.prepOutfile(filename, nbits=32)
-        self.tofile(outfile)
+        outfile.cwrite(self)
```

Comment:
Can this be in a `with` block (as is done in `toFFTfile`)? Looking at `header.prepOutfile` I think you can still do:

```
with   self.header.prepOutfile(filename, nbits=32) as outfile:
```

Reply:
also wondering if nbits=32 should be hardcoded, or if `toFile()` should have nbits argument?

Reply:
yes, it should be in a `with` block. Fixed

Do we want to save the .fft or .tim data in types other than floats?
Otherwise no need for `nbits`.